src/conductor/client/worker/worker.py

```diff
@@ -30,13 +30,13 @@ def is_callable_return_value_of_type(callable: ExecuteTaskFunction, object_type:
 
 class Worker(WorkerInterface):
     def __init__(self,
-                 task_definition_name: str,
+                 task_definition_names: str,
```

Comment:
Renaming this field breaks backward compatibility as many callers would be passing the argument name. I think just changing the type to `Union[str, list])` should suffice which btw should be done here as well in addition to the _WorkerInterface_ signature. I am pretty sure this will break a test or two.

Reply:
Thanks -- I reverted the name change